src/sysext/sysext.c

```diff
@@ -594,18 +524,32 @@ static int merge_subprocess(Hashmap *images, const char *workspace) {
                         assert_not_reached("Unsupported image type");
                 }
 
-                r = validate_version(
```

Comment:
i'd really keep validate_version() around, even if it is now shorter than before